doc/lua_api.txt

```diff
@@ -4108,6 +4263,20 @@ The Biome API is still in an experimental phase and subject to change.
     --  ^ Uses texture (string)
         playername = "singleplayer"
     --  ^ optional, if specified spawns particle only on the player's client
+	material_type_param = 12641,
+    --  ^ optional, if specified spawns particle with 
+    --  specified material type param and disable z-buffer.
+    --  Some examples:
+    --  Default value: 0,
+    --  Additive blend: 12641,
+    --  Substractive blend: 12548,
+    --  Invert blend: 12597,
+    --  See also "Particle blend".
+    	animation = {Animation definition},
+    --	^ see above. Note, that particle and particle spawners have differences.
+	glow = 15,
+    --  ^ optional, specify particle self-luminescence in darkness. 
```

Comment:
@paramat Yes. If environment light level is maximal this value will not affect particle brightness.